data-processing/pulsar-flink-new/1.15/1.15.0.0/pulsar-flink.md

```diff
@@ -0,0 +1,447 @@
+      <td>(Required) The admin URL of the Pulsar cluster.</td>
+      <td>Yes</td>
+    </tr>
+    <tr>
+      <td><h5>catalog-auth-params</h5></td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>The authentication parameters for accessing the Pulsar cluster.</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td><h5>catalog-auth-plugin</h5></td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>The name of the authentication plugin for accessing the Pulsar cluster.</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td><h5>catalog-service-url</h5></td>
+      <td style="word-wrap: break-word;">"pulsar://localhost:6650"</td>
+      <td>String</td>
+      <td>(Required) The service URL of the Pulsar cluster.</td>
+      <td>Yes</td>
+    </tr>
+    <tr>
+      <td><h5>catalog-tenant</h5></td>
+      <td style="word-wrap: break-word;">"__flink_catalog"</td>
+      <td>String</td>
+      <td>The Pulsar tenant that stores all table information.</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td><h5>default-database</h5></td>
+      <td style="word-wrap: break-word;">"default"</td>
+      <td>String</td>
+      <td>The default database when using PulsarCatalog. It will be created if it does not exist.</td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+
+### Create PulsarCatalog
+
+You can create PulsarCatalog using any of the following ways:
+
+- SQL query statements
+- Flink [Table API](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/common/)
+
+This example shows how to create PulsarCatalog using the SQL query statements.
+
+```sql
+CREATE
+CATALOG pulsar
+  WITH (
+    'type' = 'pulsar-catalog',
+    'catalog-admin-url' = '<ADMIN_URL>',
+    'catalog-service-url' = '<SERVICE_URL>'
+);
+```
+
+This example shows how to create PulsarCatalog  using the Flink [Table API](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/common/).
+
+```java
+Catalog pulsarCatalog = new PulsarCatalog();
+tableEnv.registerCatalog("pulsar",pulsarCatalog);
+```
+
+> **Note**
+>
+> If you use the SQL Client and configure catalogs in `sql-clients-defaults.yaml` YAML file, make sure that this file has been removed due to [FLIP-163](https://cwiki.apache.org/confluence/display/FLINK/FLIP-163%3A+SQL+Client+Improvements) and this file is no longer available for Flink 1.15.
+
+## Examples
+
+This section lists some examples about the Pulsar SQL connector.
+
+### Create a table
+
+This example shows how to use the Pulsar SQL connector to create a table in a JSON encoding/decoding format.
+
+```sql
+CREATE TABLE users
+(
+  `user_id`  BIGINT,
+  `item_id`  BIGINT,
+  `description` STRING,
+  `ts`       TIMESTAMP(3) METADATA FROM 'timestamp'
+) WITH (
+    'connector' = 'pulsar',
+    'topics' = 'persistent://public/default/users',
+    'service-url' = 'pulsar://localhost:6650',
+    'admin-url' = 'http://localhost:8080',
+    'format' = 'json'
```

Comment:
Today a user complained that if not using PulsarCatalog, then they need to set the `explicit` to true. Shall we include this in the documentation here ?